.github/workflows/ci.yml

```diff
@@ -11,13 +11,13 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v1
       with:
-        node-version: '14.x'
+        node-version: '15.x'
     - uses: actions/cache@v2
       with:
         path: ~/.npm
         key: npm-${{ hashFiles('package.json') }}
     - name: npm install
-      run: npm i
+      run: npm i --legacy-peer-deps
```

Comment:
npm 7 wants matching peerDeps, but we're on React 17 and other dependencies don't list it in their peerDeps.
So for now we'll need this flag.